cvo_override.yaml

```diff
@@ -36,3 +36,18 @@
     name: prometheus-k8s
     namespace: openshift-cloud-credential-operator
     unmanaged: true
+  - kind: Deployment
+    group: apps/v1
+    name: csi-snapshot-controller-operator
+    namespace: openshift-cluster-storage-operator
+    unmanaged: true
+  - kind: Deployment
+    group: apps/v1
+    name: cluster-storage-operator
+    namespace: openshift-cluster-storage-operator
```

Comment:
The commit log does not mention this one?

Reply:
@cfergeau Does mention it.